### ISSUE TITLE
MMS2R::Media filename? should strip 

### ISSUE THREAD

**Issue by laurynas** [monde/mms2r#2]
Hi, just ran into very strange bug when media filename? returned filename with newline at the end. Email was sent from Zimbra system. Quick fix was to strip output of newline with adding following code as Rails initialializer:

```
module MMS2R
  class MMS2R::Media
    def filename_with_strip?(part)
      filename_without_strip?(part).strip
    end
    alias_method_chain :filename?, :strip
  end  
end
```

**Comment by monde** [comment]
I'll fix this in the next release, I have some smart phone updates as well.

**Comment by monde** [comment]
fixed in version 3.0.0 http://rubygems.org/gems/mms2r
